### DISCUSSION TITLE
How to achieve color highlighting when the user enters certain characters, for example, when entering {{xx}}, the curly brackets and the content in the curly brackets are highlighted

### DISCUSSION THREAD

**Discussion by yrang** [ueberdosis/tiptap#5523] [General Questions] [answered]
vue 3 project：

How to achieve color highlighting when the user enters certain characters, for example, when entering {{xx}}, the curly brackets and the content in the curly brackets are highlighted

**Comment by nperez0111** [comment, marked as answer]
You would want to look at Input Rules https://tiptap.dev/docs/editor/extensions/custom-extensions/extend-existing#input-rules
& you would wrap that content in a custom mark or something so you can identify that content as needing highlighting

**Reply by yrang** [reply]
thanks！！！
